static/app/stores/utils.tsx

```diff
@@ -0,0 +1,23 @@
+import {useEffect, useState} from 'react';
+import Reflux from 'reflux';
+
+type LegacyStoreShape = Reflux.Store & {
+  // Store must have `get` function that returns the current state
+  get: () => any;
```

Comment:
is there any type we can use here?

Reply:
the type here doesn't matter, it will use the type from the store being used. It is enforcing that the store has a get function of any kind.